LocationList.py

```diff
@@ -154,15 +216,59 @@ def shop_address(shop_id, shelf_id):
+    ("Market Guard House Adult Pot 6",                              ("Collectable",  0x4D,  0x3D, ([0x3390544],None),                        'Rupee (1)',                             ("the Market", "Market", "Pot"))),
+    ("Market Guard House Adult Pot 7",                              ("Collectable",  0x4D,  0x2A, None,                        'Rupee (1)',                             ("the Market", "Market", "Pot"))),
+    ("Market Man in Green House Pot 1",                             ("Collectable",  0x2B,  0x21, None,                        'Recovery Heart',                        ("the market", "Market", "Pot"))),
+    ("Market Man in Green House Pot 2",                             ("Collectable",  0x2B,  0x22, None,                        'Recovery Heart',                        ("the market", "Market", "Pot"))),
+    ("Market Man in Green House Pot 3",                             ("Collectable",  0x2B,  0x23, None,                        'Recovery Heart',                        ("the market", "Market", "Pot"))),
+    
     # Hyrule Castle
     ("HC Malon Egg",                                                 ("NPC",          0x5F,  0x47, None,                        'Weird Egg',                             ("Hyrule Castle", "Market",))),
     ("HC Zeldas Letter",                                             ("NPC",          0x4A,  0x0B, None,                        'Zeldas Letter',                         ("Hyrule Castle", "Market",))),
     ("HC Great Fairy Reward",                                        ("Cutscene",     0xFF,  0x11, None,                        'Dins Fire',                             ("Hyrule Castle", "Market", "Fairies"))),
     ("HC GS Tree",                                                   ("GS Token",     0x0E,  0x04, None,                        'Gold Skulltula Token',                  ("Hyrule Castle", "Skulltulas",))),
     ("HC GS Storms Grotto",                                          ("GS Token",     0x0E,  0x02, None,                        'Gold Skulltula Token',                  ("Hyrule Castle", "Skulltulas", "Grottos"))),
+    ("HC Storms Grotto Pot 1",                                       ("Collectable",  0x3E,  0x22, None,                        'Recovery Heart',                        ("Hyrule Castle", "Grottos", "Pot"))),
+    ("HC Storms Grotto Pot 2",                                       ("Collectable",  0x3E,  0x23, None,                        'Recovery Heart',                        ("Hyrule Castle", "Grottos", "Pot"))),
+    ("HC Storms Grotto Pot 3",                                       ("Collectable",  0x3E,  0x24, None,                        'Recovery Heart',                        ("Hyrule Castle", "Grottos", "Pot"))),
+    ("HC Storms Grotto Pot 4",                                       ("Collectable",  0x3E,  0x25, None,                        'Recovery Heart',                        ("Hyrule Castle", "Grottos", "Pot"))),
```

Comment:
I see the rupees you get for shooting the torches aren't shuffled, is that from hacking difficulty or just too obscure? (Either reasoning is acceptable, lol)

Reply:
I just never went about hacking the wonderitem actors which spawn those rupees. It could theoretically be done.

Reply:
Makes sense

Reply:
One of the issues in particular with wonderitems is that they set a switch flag when the wonderitem is trigger as opposed to when the collectible is collected. This could cause softlocks if you fail to collect the item. Example of this is the sign in KF that drops a rupee when you slash it.